### DISCUSSION TITLE
Status of rq-scheduler?

### DISCUSSION THREAD

**Discussion by blalor** [rq/rq#2072] [Q&A]
What's the state and status of rq-scheduler as it relates to rq 1.16.1?  I'm building a new application around rq and am going to need to have some tasks periodically invoked.  It seems that rq now has most if not all of what's needed to support scheduled tasks built in, and the missing piece is something to enqueue the tasks when they're needed.  Is there a benefit to using rq-scheduler over rolling my own with something like apscheduler?

**Comment by amks1** [comment]
I've been using rq with rq-scheduler since 2021. Works ok so far.
